cmd/down.go

```diff
@@ -42,6 +43,10 @@ func Down() *cobra.Command {
 		RunE: func(cmd *cobra.Command, args []string) error {
 			log.Info("starting down command")
 
+			if k8Client.InCluster() {
```

Comment:
`down` works if your dev environment is in Okteto Cloud... same for `exec`...
Maybe we shouldn't inject the `okteto` and `kubectl` binary if you are not in Okteto Cloud.

Reply:
yes, I think that's probably a better approach.